docs/scripts/doc8_style_check.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# halt script on error
+set -e
+# Check for Style Code Violations
+doc8 --max-line-length 100 source/scancode-toolkit --ignore D000 --quiet
```

Comment:
Dont we want to run doc8 over everything, not the just `scancode-toolkit` docs?

Reply:
Also, can you add a comment about what `d000` is? We ignore it here which is probably fine, it would be nice to know what exactly this value means in a comment so future maintainers know what is going on.

Reply:
On `D000`, it's basically checking for invalid RST formats, which is already checked in sphinx-build. Additionally, it generates errors on `include` statements as they aren't located in the same folder, which was required for cross-section redundancy removal. I'll see if I can remove this, but I didn't think this was important so I left it on. [Here](https://aboutcode.readthedocs.io/en/latest/scancode-toolkit/contribute/contrib_doc.html#what-is-checked) is the docs on this. I'll leave a comment, sure.

Reply:
On running doc8 over everything, it just means fixing a lot of (500+) doc style errors all over the other docs. I have it in my To-Do list, I just thought I'd finish migration work first. I'll open another PR soon on this issue in a couple of days, does that work?

Reply:
@AyanSinhaMahapatra I think that is fine. However, can you open a new ticket or tickets in this repo so we can track this and have some record of the tool docs that need to be cleaned up and added to the CI checks?

And your analysis of the `D000` stuff is good enough for now. We will probably want to update this later, but for now we can leave it in I think.

Reply:
@majurg I'm actually working on this now, I'll create another PR in an hour and change this to `source/`.

Reply:
Added this [commit](https://github.com/nexB/aboutcode/pull/29/commits/ebe950b3fe8672d9d3134384992b9b464ed2cfa3) instead fixing all style Errors. This [commit](https://github.com/nexB/aboutcode/pull/30/commits/9b17dc0da08ecfd23a269fb73f9177bf5f462f06) makes the `doc8` tests run for all docs.